modules/Kconfig.tfm

```diff
@@ -78,6 +78,28 @@ config TFM_PROFILE
 	  profile_medium and profile_small. The default profile does not need
 	  to have this configuration set.
 
+choice
+	bool
+	prompt "The build type sent to the TFM build system"
```

Comment:
Just did a UI try of this:
![image](https://user-images.githubusercontent.com/9802583/107504780-179a7200-6b9c-11eb-8f41-f69e64dd8a01.png)

Maybe shorten this to:
![image](https://user-images.githubusercontent.com/9802583/107504974-53cdd280-6b9c-11eb-8ca6-5035c195b905.png)

```suggestion
	prompt "TFM build type"
```